.internal/ansible/ansible_collections/internal/gha_docs/roles/generate/tasks/main.yml

```
@@ -3,6 +3,12 @@
   set_fact:
     _src: "{{ lookup('ansible.builtin.file', file) | from_yaml }}"
 
+- name: Ensure output dir exists
+  ansible.builtin.file:
+    path: '{{ output | dirname }}'
+    state: directory
+    mode: '744'
```

Ah, one thing: shouldn't this be `755`? Or is it intentional that the executable bit is missing, while reading is allowed?
```suggestion
    mode: '755'
```

---

I would think we don't need the executable bit, the directories will contain markdown files. Thought it's entire;y possible I'm missing something...

---

You cannot access any files in the folder if the folder has no executable bit. Basically only `700` or `755` make sense.

---

Ah, I forgot about the meaning of X bit in dirs. 744 worked fine in testing (within the workflow) but perhaps locally that would be problematic.

---

thank you @felixfontein good catch, and thanks for reviewing again!